gravitee-apim-console-webui/src/portal/customization/top-bar/menu-link-edit/menu-link-edit.component.scss

```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright (C) 2021 The Gravitee team (http://gravitee.io)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+@use '../../../../scss/gio-layout' as gio-layout;
+@use 'sass:map';
+@use '@angular/material' as mat;
+@use '@gravitee/ui-particles-angular' as gio;
+
+$typography: map.get(gio.$mat-theme, typography);
+
+:host {
+  @include gio-layout.gio-responsive-content-container;
+}
+
+.menu-link-edit {
+  &__title {
+    padding: 20px 24px;
+    border-bottom: 1px mat.m2-get-color-from-palette(gio.$mat-dove-palette, 'darker10') solid;
+
+    & > h3 {
+      margin: 0;
+    }
+
+    & > h3 > span {
+      margin-left: 12px;
+    }
```

Comment:
I wonder if this could be simplified with just `flex`? For just having the `<h3>` tag as the title, I don't see why it needs so much padding + margin adjustment.